src/api/config/initializers/db_query_matchers.rb

```diff
@@ -0,0 +1,15 @@
+DBQueryMatchers.configure do |config|
+  config.ignores = [
+    # Executed multiple times when fetching something in the configuration
```

Comment:
BTW: can we stop that? I have a branch somewhere to cache this as it's just very annoying (in the debug logs)

Reply:
I don't know why we're storing configuration in the database. Rails can be configured with custom configuration as explained in [Configuring Rails Applications](https://guides.rubyonrails.org/configuring.html#custom-configuration). This is what I would do to refactor this.

Reply:
Because there is a UI to change it